python/submitThem_old.py

```diff
@@ -607,7 +607,7 @@ def appendList(): optionsList.append({'cut':copy(_cut),'var':copy(_treevar),'nam
         if shape__.replace(' ','')  == '': continue#to avoid && case
         shapecut_split_ = shape__.split('||')
         for shape_ in  shapecut_split_:
-            new_cut_list=sys_cut_suffix[syst]
```

Comment:
so we still need to differentiate the submitThem,py and submitThem_old? probably not

Reply:
(once more, not necessarily needs to be addressed in this PR. we can probably start using issues to track things to be changed)

Reply:
(issues need to be activated in forked repositories, just go to settings and then tick on "Issues"

Reply:
@perrozzi I agree concerning the dataset folder. We can put it in Xbb/python.

Reply:
@perrozzi I agree too.